### PR TITLE
added: cg-proc wrapper

### PR DESCRIPTION
**Pull request by Vaydheesh**
- use updated python wrappers from package builds

### CODE REVIEW

File: apertium/utils.py [hunk 1]
```diff
@@ -86,6 +87,11 @@ def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
             elif 'apertium-tagger' == command[0]:
                 command += [input_file.name, output_file.name]
                 apertium_core.ApertiumTagger(len(command), command)
+            elif 'cg-proc' == command[0]:
+                dictionary_path = command[-1]
+                obj = constraint_grammar.CGProc(dictionary_path)
```
Comment (sushain97) [inline apertium/utils.py]: Please use more specific names. e.g. cg here.

File: tests/__init__.py [hunk 2]
```diff
@@ -107,3 +107,9 @@ def test_translator_en_spa(self):
     def test_en_spa(self):
         translated = apertium.translate('eng', 'spa', 'cats')
         self.assertEqual(translated, 'Gatos')
+
+    def test_kaz_tat(self):
+        apertium.installer.install_module('kaz-tat')
```
Comment (sushain97) [inline tests/__init__.py]: Is cg-conv being installed on Windows?

Comment (Vaydheesh) [reply tests/__init__.py]: Translation requires `cg-proc`, and yes `cg3` binaries are available in `apertium-all-dev`

Comment (sushain97) [reply tests/__init__.py]: :+1:

**Pull request merged**